### DISCUSSION TITLE
How to plot GOS vs. Frequency?

### DISCUSSION THREAD

**Discussion by SingPr** [mtex-toolbox/mtex#1542] [Ask Anything] [answered]
Dear community,
     Here is a question -I can plot the GOS maps, but how to plot GOS vs. Frequency like below?
![image](https://user-images.githubusercontent.com/89964907/188399303-b1f72826-c494-4c01-bf28-950098dcfdf9.png)
![image](https://user-images.githubusercontent.com/89964907/188399361-b6d3104b-63ef-4ea8-8b27-df8dc70a6151.png)
Does anyone know how to fix this? Thanks very much!

Best regards,
A naive phd

**Comment by zmichels** [comment, marked as answer]
do you mean something like:
```matlab
histogram(grains.GOS./degree,50)
```

**Reply by SingPr** [reply]
thanks! It works.

**Reply by ddyyasfh** [reply]
Please how to export frequencies to txt format

**Reply by ddyyasfh** [reply]
How to export the line chart below？

**Reply by ralfHielscher** [reply]
check out https://de.mathworks.com/help/matlab/ref/histcounts.html 

Ralf.

**Reply by ddyyasfh** [reply]
Thank you so much
